### PR TITLE
Improve seeds to have more investments per investor

### PR DESCRIPTION
**Pull request by kleyv**
I changed the algorithm to have more investors and so that each
investor can have more than one investment and bookmark per
project.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -50,38 +50,48 @@
...
-User.where(role: 'investor').each do
-  Funding.create!(
-    funding_amount: (0..10_000).to_a.sample,
-    funding_shares: (5..100).to_a.sample,
-    project_id: Project.all.sample.id,
-    investor_id: User.where(role: 'investor').sample.id,
-    accepted?: [true, false].sample
+puts 'Generating fundings...'
+User.where(role: 'investor').each do |investor|
+  rounds = (1..Project.count).to_a.sample
```
Comment (raffoz) [inline db/seeds.rb:71]: rounds = rand(1..Project.count)

File: db/seeds.rb [hunk 2]
```diff
@@ -50,38 +50,48 @@
...
+  for i in (1..rounds).to_a do
+    Funding.create!(
+      funding_amount: (1..10_000).to_a.sample,
+      funding_shares: (1..100).to_a.sample,
+      project_id: Project.all[i-1].id,
+      investor_id: investor.id,
+      accepted?: [true, false].sample
```
Comment (raffoz) [inline db/seeds.rb:78]: .to_a is not necessary
instead of .sample on array, do rand() on integers

File: db/seeds.rb [hunk 3]
```diff
@@ -50,38 +50,48 @@
...
-  Bookmark.create!(
-    investor_id: User.where(role: 'investor').sample.id,
-    project_id: Project.all.sample.id
+puts 'Generating bookmarks...'
+User.where(role: 'investor').each do |investor|
+  rounds = (1..Project.count).to_a.sample
+  for i in (1..rounds).to_a do
+    Bookmark.create!(
+      investor_id: investor.id,
+      project_id: Project.all[i-1].id,
```
Comment (raffoz) [inline db/seeds.rb:92]: .to_a is not necessary
instead of .sample on array, do rand() on integers

Comment (raffoz) [reply db/seeds.rb:92]: -1 is still needed if not array?

File: db/seeds.rb [hunk 4]
```diff
@@ -14,7 +14,7 @@
 
 # Add 10 users
```
Comment (raffoz) [inline db/seeds.rb:15]: 20 users

File: db/seeds.rb [hunk 5]
```diff
@@ -32,8 +32,8 @@
 puts 'Investors done!'
 
 # Add 5 owners
```
Comment (raffoz) [inline db/seeds.rb:34]: 15 owners

**Review by raffoz — CHANGES_REQUESTED** [review]

File: db/seeds.rb (hunk 3 above)
Comment (kleyv) [reply db/seeds.rb:92]: The -1 is so that it starts with the project at index 0 instead of index 1

**Review by raffoz — APPROVED** [review]

**Pull request merged**